cmd/soroban-rpc/internal/config/options.go

```diff
@@ -212,24 +215,36 @@ func (cfg *Config) options() Options {
 			ConfigKey:    &cfg.CheckpointFrequency,
 			DefaultValue: uint32(64),
 		},
+		{
+			Name: "history-retention-window",
+			Usage: fmt.Sprintf(
+				"configures history retention window for transactions and events, expressed in number of ledgers,"+
+					" the default value is %d which corresponds to about 24 hours of history",
+				OneDayOfLedgers),
+			ConfigKey:    &cfg.HistoryRetentionWindow,
+			DefaultValue: uint32(OneDayOfLedgers),
+			Validate:     positive,
+		},
+		// TODO: remove
 		{
 			Name: "event-retention-window",
 			Usage: fmt.Sprintf(
-				"configures the event retention window expressed in number of ledgers,"+
+				"(Deprecated, overidden by history-retention-window) configures the event retention window expressed in number of ledgers,"+
```

Comment:
typo:
```suggestion
				"(Deprecated, overridden by history-retention-window) configures the event retention window expressed in number of ledgers,"+
```

Reply:
Also it's not overridden, right? It's the max of the three. Maybe this is okay as help text, though, so that we can move users away from setting that flag.

Reply:
Yeah correct it is still max of 3, but we want users to just set the `history-retention-window `